spec/support/pages/projects/index.rb

```diff
@@ -671,14 +541,6 @@ def boolean_filter?(filter)
         %w[active member_of favored public templated].include?(filter.to_s)
       end
 
-      def autocomplete_filter?(filter)
-        filter.has_css?('[data-filter-autocomplete="true"]', wait: 0)
-      end
-
-      def date_filter?(filter)
-        filter[:"data-filter-type"] == "date"
-      end
-
```

Comment:
👍  for cleaning up the duplicated methods. I wasn't aware of those. How about we open a code maintenance ticket to move even more over to the module, e.g. `date_time_filter?` but also the `set_advanced_filter` method?

Reply:
Done: https://community.openproject.org/wp/63111